tests/regression/46-apron2/81-overflow-caching.c

```diff
@@ -0,0 +1,14 @@
+// SKIP PARAM: --set ana.activated[+] apron --set ana.relation.privatization top
+ 
+#include <pthread.h>
+#include <goblint.h>
+
+    int num = 1;
+
+
+int main() {
+
+    while(num > 0)
+        num++; // Here num overflows
+    return 0;
+}
```

Comment:
This should contain a check that the end of the loop is in fact reachable, otherwise this test will pass even when the problem reappears.

Reply:
indeed. fixed now